packages/loader/container-definitions/src/loader.ts

```diff
@@ -259,6 +259,11 @@ export enum LoaderHeader {
      * otherwise, version sha to load snapshot
      */
     version = "version",
+
+    /**
+     * Max time container will wait for a leave message of a disconnected client.
+     */
+    maxClientLeaveWaitTime = "maxClientLeaveWaitTime",
```

Comment:
don't add this as a header, add it as an option

Reply:
Added.